plugins/tiletypes.cpp

```diff
@@ -722,27 +745,40 @@ command_result executePaintJob(color_ostream &out)
     }
 
     CoreSuspender suspend;
-    uint32_t x_max = 0, y_max = 0, z_max = 0;
-    int32_t x = 0, y = 0, z = 0;
 
     if (!Maps::IsValid())
     {
         out.printerr("Map is not available!\n");
         return CR_FAILURE;
     }
+
+    uint32_t x_max = 0, y_max = 0, z_max = 0;
     Maps::getSize(x_max, y_max, z_max);
 
-    if (!Gui::getCursorCoords(x,y,z))
+    DFCoord cursor;
```

Comment:
I might use `df::coord` here just because it's the newer name (and maybe we should deprecate DFCoord...)

Reply:
is it? from its usage in Maps, I had thought that DFCoord was the preferred spelling. I'll use `df::coord` in the future.

Reply:
In general, names in the `df::` namespace are newer than others (like `t_engraving` in Engravings.h). From a [blame](https://github.com/DFHack/dfhack/blame/3a7306eb03dc9504e7f2e0a5f0ad688d8c23013d/library/include/modules/Maps.h#L93), DFCoord appears to have been [replaced](https://github.com/DFHack/dfhack/commit/cc510a2c4b9887b885c939419bf67147ee997ac6) a while ago, but evidently the old name stuck around in some places (and maybe was even reintroduced in other new code). Hard to say. Since there are so few external plugins these days, we could probably stand to deprecate and eventually remove the old names for things now.

Reply:
Ack. For my part, I was using it because the Maps API used it, and because it was slightly shorter : )